geniza/entities/templates/entities/snippets/person_option_label.html

```diff
@@ -0,0 +1,80 @@
+{# template snippet for displaying person label on a form #}
+{# used on person merge form to provide enough information to merge accurately #}
+
+<div class="merge-document-label">
+    <h2>{{ person }}
+        <a target="_blank" href="{% url 'admin:entities_person_change' person.pk %}"
+           title="Go to this person's admin edit page">
+            <img src="/static/admin/img/icon-changelink.svg" alt="Change"></a>
+    </h2>
+    {% if person.names.count %}
+        {% for name in person.names.all %}
+            <div class="form-row">
+                <label>{% if name.primary %}Display n{% else %}N{% endif %}ame ({{ name.language }})</label><div>{{ name }}</div>
+            </div>
+        {% endfor %}
+    {% endif %}
+    {% if person.role %}
+        <div class="form-row">
+            <label>Social role</label><div>{{ person.role }}</div>
+        </div>
+    {% endif %}
+    {% if person.gender %}
+        <div class="form-row">
+            <label>Gender</label><div>{{ person.gender }}</div>
+        </div>
+    {% endif %}
+    {% if person.description %}
+        <div class="form-row">
+            <label>Description</label><div>{{ person.description }}</div>
+        </div>
+    {% endif %}
```

Comment:
does this display all languages or only current language? should it display all, or all that have any content? wonder if there is a way to use model translate tabs to embed them, but I think then users would have to click to see if there's any content

Reply:
It will display only the current language, which is how the document merge currently works. I spent a bit of time trying to figure out how to loop through the different language options, but it's not trivial; we could hard code English, Arabic and Hebrew... But since it's already not working like this on the document merge, and it seems quite unlikely that a person would have a Hebrew or Arabic description but _not_ an English one, I think it's probably ok for now. (And we can add as a feature later if it ends up being needed.)

Reply:
I see. That sounds reasonable. The main thing is that we're not losing any content in any language when merging, and it sounds like we handle that in both cases.

This seems like a feature that modeltranslate ought to offer, but I just looked at the docs and don't see anything like it. I'm having trouble figuring out how to articulate what we need to even find if there's a relevant issue.